application-tour-ui/src/main/resources/TourCode/TourJson.xml

```diff
@@ -132,6 +147,5 @@
   #set ($discard = $response.setContentType('application/json'))
   $jsontool.serialize({'tours': $tours})
 #end
-{{/velocity}}
-</content>
+{{/velocity}}</content>
```

Comment:
On my side, my IDE allow me to revert some *part* of the files, and I know there is a git command for that too.